dashboard/db/migrate/20170822231226_create_pd_regional_partner_contacts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePdRegionalPartnerContacts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_regional_partner_contacts do |t|
+      t.references :user, null: false, index: true
+      t.references :regional_partner, null: true, index: true
```

Comment:
Should either of these indexes be unique? Should the combination be unique? It would have been nice to see some comments on what the fields mean. Is the `RegionalPartner` the contact? What is the `User`? Etc.